cmd/swapcli/main.go

```diff
@@ -620,33 +619,56 @@ func runGetPastSwapIDs(ctx *cli.Context) error {
 	}
 
 	fmt.Println("Past swap offer IDs:")
-	for i, id := range ids {
-		fmt.Printf("%d: %s\n", i, id)
-	}
 	if len(ids) == 0 {
 		fmt.Println("[none]")
+		return nil
 	}
+
+	for i, id := range ids {
+		if i > 0 {
+			fmt.Printf("---\n")
+		}
+
+		fmt.Printf("ID: %s\n", id.ID)
+		fmt.Printf("Start time: %s\n", id.StartTime)
+		fmt.Printf("End time: %s\n", id.EndTime)
+	}
+
 	return nil
 }
 
 func runGetOngoingSwap(ctx *cli.Context) error {
 	offerID := ctx.String(flagOfferID)
 
 	c := newRRPClient(ctx)
-	info, err := c.GetOngoingSwap(offerID)
+	resp, err := c.GetOngoingSwap(offerID)
 	if err != nil {
 		return err
 	}
 
-	receivedCoin := "ETH"
-	if info.Provided == coins.ProvidesETH {
-		receivedCoin = "XMR"
+	fmt.Println("Ongoing swaps:")
+	if len(resp.Swaps) == 0 {
+		fmt.Println("[none]")
+		return nil
 	}
 
-	fmt.Printf("Provided: %s %s\n", info.ProvidedAmount.Text('f'), info.Provided)
-	fmt.Printf("Receiving: %s %s\n", info.ExpectedAmount, receivedCoin)
-	fmt.Printf("Exchange Rate: %s ETH/XMR\n", info.ExchangeRate)
-	fmt.Printf("Status: %s\n", info.Status)
+	for i, info := range resp.Swaps {
+		if i > 0 {
+			fmt.Printf("---\n")
+		}
+
+		receivedCoin := "ETH"
+		if info.Provided == coins.ProvidesETH {
+			receivedCoin = "XMR"
+		}
+
+		fmt.Printf("ID: %s\n", info.ID)
+		fmt.Printf("Start time: %s\n", info.StartTime)
```

Comment:
That is going to be a pretty ugly string. Do we want to use `.Format(common.TimeFmtSecs)` or something similar?

Reply:
Whatever you use, use them on the other places we print start/end times above and below in this file.